### ISSUE TITLE
REST module 'execute()' prefixes slash to url which already have leading slash

### ISSUE THREAD

**Issue by nicholascus** [Codeception/Codeception#5473]
#### What are you trying to achieve?
Consistency in URL construction when both PhpBrowser and REST module are used.

#### What do you get instead?

Requests sent with REST module get leading slash doubled. So server get log full of
192.168.176.9 - - [17/Apr/2019:01:05:58 +1000] "POST //

```yml
actor: FunctionalTester
modules:
    enabled:
        - REST:
            url: 'http://%HOST%:%PORT%/'
            depends: PhpBrowser
        - \Helper\Functional
        - PhpBrowser:
            url: 'http://%HOST%:%PORT%/'
        - Asserts
```

**Comment by Naktibalda** [comment]
The patch was inconsistent with bug report and it looks like it didn't fix anything.
Example of configuration above uses absolute urls starting with `http://`,
but the code only prepends `/` if there is no `://` in configured url - `if (strpos($url, '://') === false) {`

Provided patch checks the same byte in a different way which makes no difference.

```php
$url = $this->config['url'] . $url;
strpos($url, '/') !== 0 && $this->config['url'][0] !== '/'
```

**Comment by nmihaiv** [comment]
If the config url doesn't contain http://, then the request link will have a / in front of it.

**Comment by Naktibalda** [comment]
Fixed by #5511
